src/internal/internalAPIs.ts

```diff
@@ -62,6 +63,13 @@ function handleChangeSettings(): void {
   }
 }
 
+function handleGetLogRequest(): void {
+  if (GlobalVars.getLogHandler) {
+    const log: string = GlobalVars.getLogHandler();
+    sendMessageRequest(GlobalVars.parentWindow, "sendLog", [log]);
```

Comment:
We should create a separate function call 
export function sendLog(log: String) {
          sendMessageRequest(GlobalVars.parentWindow, "sendLog", [log]);
 }

that can be use by app to send log to teams client. getLogHandler should call microsoftTeams.sendLog function to send call.

Reply:
i had the method logMessages earlier.
Yuri's suggestion above was to be able to send logs only when asked from app